README.md

```diff
@@ -19,9 +19,9 @@ Project Bonneville was research aimed at determining best approaches to enabling
 
 ## Installing
 
-Once built, the result can be installed with the following command. This is just an example - see the install help output for all options, and a slightly more indepth example [here](usage.md):
+Once built, the result can be installed with the following command. This is just an example - see the vic-machine help output for all options, and a slightly more indepth example [here](usage.md):
 ```
-install.sh -g -t '<user>:<password>@<target-host>' -i <datastore-name> <vch-name>
+vic-machine -target target-host -iStore <datastore-name> -name <vch-name> -user <user> -passwd <password> -pool <resource pool path in govc format> -gen
```

Comment:
Please do use the longer names, i.e. -image-store instead of -iStore. It's significantly more self-documenting so long as the terms map precisely to the actual VIC documentation.

Regarding -pool... I'm unsure whether we should call that `pool`, `compute`, or `compute-resource` but I am _certain_ that I don't want to end up with the -pool=x -host=y -cluster=z -dc=agh mess that we had with bonneville. The compute resource (given that's the vmodl name) will be specified by a single flag unless there is a __very__ good reason to add others.